app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -106,16 +102,12 @@ open class BpkSnapshotTest {
       scenario.onActivity { activity ->
         with(activity) {
           setContent {
-            BpkTheme {
-              CompositionLocalProvider(*providers) {
-                Surface(
-                  modifier = Modifier
-                    .size(size.width.dp, size.height.dp)
-                    .background(background.takeOrElse { BpkTheme.colors.background }),
-                  content = { content() },
-                )
-              }
-            }
+            BackpackPreview(
+              modifier = Modifier.size(size.width.dp, size.height.dp),
+              background = background,
+              providers = providers,
+              content = { content() }
```

Comment:
We can just pass the content here without lambda :) Also, could you please add a trailing comma?